Update Angular docs and samples after removing 'divarea' editor type (#325)
ckeditor/ckeditor4-docs: docs/guide/dev/integration/angular/README.md

></ckeditor>
```

> ### Using Div Area Editor Type

Reviewer: ```suggestion
### Using the Div-based Editor Type
```

Before the [`2.0.0` integration version](https://www.npmjs.com/package/ckeditor4-angular/v/2.0.0) there was also a separate `divarea` editor type (an {@link guide/dev/inline/README inline editor} with a {@link features/uitypes/README#fixed-ui-for-inline-editor fixed UI}).

> To use the `divarea` editor in `2.0.0` and later versions you just need to include a [Div Editing Area plugin](https://ckeditor.com/cke4/addon/divarea) in the editor config:

Reviewer: ```suggestion
To use the `divarea` editor in `2.0.0` and later versions, you just need to include the [Div Editing Area](https://ckeditor.com/cke4/addon/divarea) plugin in the editor configuration:
```

```html
<ckeditor
	data="<p>Some initial data</p>"
	type="inline"
	[config]="{ extraPlugins: 'divarea' }">
></ckeditor>
```

> To use the `divarea` editor in versions prior `2.0.0` (up to `1.3.0`), you have to change editor type to `divarea`:

Reviewer: ```suggestion
To use the `divarea` editor in versions prior `2.0.0` (up to `1.3.0`), you have to change the editor type to `divarea`:
```
